src/ooxml/java/org/apache/poi/xssf/extractor/XSSFImportFromXML.java

```diff
@@ -160,6 +172,57 @@ public void importFromXML(String xmlInputString) throws SAXException, XPathExpre
             }
         }
     }
+    
+    /**
+     * Resize the table by setting a new row count.
+     * 
+     * If the new row count is less than the current row count, superfluous rows will be cleared.
+     * If the new row count is greater than the current row count, cells below the table will be overwritten by the table.
+     *
+     * @param table the table to modify
+     * @param newRowCount new row count for the table
+     */
+    private void resizeTable(XSSFTable table, int newRowCount) {
+        
+        table.updateReferences();
+        int rowCount = table.getRowCount();
+        if(rowCount == newRowCount) {
```

Comment:
The project uses space between keywords (if, for, try, while) and opening parenthesis.